src/index.html

```diff
@@ -1,13 +1,61 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css" >
+  </head>
+  <body>
+    <div class="container">
+      <div class="card container__card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          class="card__product-img"
+        >
+
+        <h1 class="card__product-name">
```

Comment:
```suggestion
        <h2 class="card__product-name">
```
h1 should be 1 per whole page
what will be in catalog?